package.json

```diff
@@ -516,7 +528,39 @@
                     "name": "Targets"
                 }
             ]
-        }
+        },
+        "viewsWelcome": [
+            {
+                "view": "vscode-edge-devtools-view.targets",
+                "contents": "Launch an instance of Microsoft Edge to begin inspecting and modifying webpages.\n[Launch Instance](command:vscode-edge-devtools-view.launch)",
+                "when": "titleCommandsRegistered && launchJsonStatus != Supported"

```

Comment:
We should show all the content for the empty target list immediately and not wait on the `titleCommandsRegistered ` flag.  We can disable the buttons that actually launch commands until they're ready with the `enablement` flag when the commands are registered.